baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/Slf4jLogsafeArgs.java

```diff
@@ -66,7 +66,7 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
                 state) ? 2 : 1;
         int endArg = ASTHelpers.isCastable(
                 ASTHelpers.getType(allArgs.get(lastIndex)),
-                state.getTypeFromString("java.lang.Exception"),
+                state.getTypeFromString("java.lang.Throwable"),
```

Comment:
can you add a test (or change an existing one) to cover this?

Reply:
Sure

Reply:
Done